src/components/virtual-scroll/virtual-for/virtual-for.component.less

```diff
@@ -0,0 +1,8 @@
+[uxVirtualForContainer] {
+    .ux-virtual-scroll-spacer {
+        visibility: hidden;
+        border: none !important;
+        margin: 0 !important;
+        padding: 0 !important;
```

Comment:
What was the reason for using !important here? If these are essential for functionality rather than presentation then perhaps it would better to apply them directly to the elements as `style.border` etc.